packages/react/src/components/Table/TableToolbar/TableToolbar.jsx

```diff
@@ -138,12 +139,27 @@ const propTypes = {
     customToolbarContent: PropTypes.node,
     /** available batch actions */
     batchActions: PropTypes.arrayOf(
-      PropTypes.shape({
-        id: PropTypes.string.isRequired,
-        labelText: PropTypes.string.isRequired,
-        icon: PropTypes.node,
-        iconDescription: PropTypes.string,
-      })
+      PropTypes.oneOfType([
```

Comment:
Are these two shapes exact copies of any other props in the table? If so perhaps we can defined them in a single location

Reply:
not exact, but I did make some re-usable objects to clean it up a bit.